src/pages/device-list/device-list.ts

```diff
@@ -161,4 +179,37 @@ export class DeviceList {
   doRefresh(refresher) {
     this.listDevices(refresher);
   }
+
+  setConnectionString() {
+    let enterConnectionString = this.alertCtrl.create({
+      title: 'IoT Hub Connection String',
+      inputs: [
+        {
+          name: 'iotHubConnectionString',
+          placeholder: 'IoT Hub Connection String'
```

Comment:
Same here. If user has already set iotHubConnectionString, should display the value that user sets.